### DISCUSSION TITLE
why is jdk-threading-plugin in bootstrap-plugins?

### DISCUSSION THREAD

**Discussion by lingli0219** [apache/skywalking#8784] [General]
I wonder why jdk-threading-plugin is in bootstrap-plugins. Why is it dangerous to use? And after setting plugin.jdkthreading.threading_class_prefixes for specific usage, is it still dangerous?

**Comment by wu-sheng** [comment]
We don't know whether it is dangerous for you. But instrumenting JDK should be careful always. You could have many ways to use those APIs, we don't know how.

**Reply by lingli0219** [reply]
OK. Thank you for your reply.
